src/test/fixtures/custom/my-class.js

```diff
@@ -1,3 +1,9 @@
+/**
+ * @template T
+ * @constructor
+ */
+function MyParameterizedType() {}
```

Comment:
probably pointless at the min seeing as we don't check if named types exist or not

Reply:
Yeah I would just drop this, not necessary.